src/ploneorg/addonlisting/models/addon.xml

```diff
@@ -78,12 +78,12 @@
       <fieldset name="pypi_data"
                 label="PyPI Data"
                 description="Data in this tab will be automatically updated from PyPI."
-                form:mode="z3c.form.interfaces.IAddForm:hidden"
+                
```

Comment:
Why do you remove the form marker?
is this for testing purpose? then maybe a comment would have done it also.